### ISSUE TITLE
can't edit the wiki

### ISSUE THREAD

**Issue by hassanzadeh** [facebook/react#12104]
Hello,
I can't edit the wiki to add my website to the list there, is editing disabled?

**Comment by gaearon** [comment]
I don't think so, can you post a screenshot/

**Comment by samdenty99** [comment]
Public wiki editing has been turned off =>

![image](https://user-images.githubusercontent.com/13242392/37134954-d215cc34-2292-11e8-91c4-ff7ab82f0357.png)

https://github.com/facebook/react/wiki/Sites-Using-React/_edit

**Comment by ppsreejith** [comment]
Hey, so does anyone know how one can contribute to the wiki?

**Comment by hrdymchl** [comment]
Yup, getting the same screenshot as @samdenty99, and would love to contribute our project to the wiki too!
